docs/en/ingest-management/elastic-agent/configuration/examples/config-file-example-apache.asciidoc

```diff
@@ -0,0 +1,126 @@
+[[config-file-example-apache]]
+= Config file example: Apache HTTP Server
+
+++++
+<titleabbrev>Apache HTTP Server</titleabbrev>
+++++
+
+Include these settings in your standalone {agent} `elastic-agent.yml` configuration file to ingest data from Apache HTTP server. 
+
+* <<config-file-example-apache-logs>>
+* <<config-file-example-apache-metrics>>
+
+[[config-file-example-apache-logs]]
+== Apache HTTP Server logs
+
+["source","yaml"]
+----
+outputs: <1>
+  default:
+    type: elasticsearch <2>
+    hosts:
+      - 'https://94d343762a2b465f976aa292df6a0fe1.us-central1.gcp.cloud.es.io:443'
```

Comment:
I believe this url here should be made generic and not be pointing to an actual URL. Somethign like "your elasticsearch output" should suffice.

Reply:
I've changed this one to `'{elasticsearch-host-url}' <3>` and the description to `<3> The URL of the Elasticsearch cluster where output should be sent, including the port number. For example https://12345ab6789cd12345ab6789cd.us-central1.gcp.cloud.es.io:443.`